src/VirtoCommerce.CustomerModule.Data/Handlers/IndexMemberChangedEventHandler.cs

```diff
@@ -46,10 +55,21 @@ public async Task TryIndexMemberBackgroundJob(string[] indexMemberIds)
             await TryIndexMember(indexMemberIds);
         }
 
+        [DisableConcurrentExecution(60 * 60 * 24)]
+        public async Task TryDeleteIndexMemberBackgroundJob(string[] indexMemberIds)
+        {
+            await TryDeleteIndexMember(indexMemberIds);
```

Comment:
`async` and `await` can be removed:
`return TryDeleteIndexMember(indexMemberIds);`
The same for other methods.